src/js/binaryen.js-post.js

```diff
@@ -2158,6 +2158,28 @@ function wrapModule(module, self) {
       );
     });
   };
+  self['getFunctionTable'] = function() {
+    return {
+      'imported': Boolean(Module['_BinaryenIsFunctionTableImported'](module)),
+      'segments': (function() {
+        var arr = [];
+        for (var i = 0, j = Module['_BinaryenGetNumFunctionTableSegments'](module) ; i !== j ; ++i)
```

Comment:
It would be good to give the end variables more meaningful names like `numSegments` and only use single-letter names for loop indices.